src/assets/data/defaultConfig.ts

```diff
@@ -134,22 +131,39 @@ export const DefaultEndPoint: string =
   'https://radar-cns-platform.rosalind.kcl.ac.uk/'
 // export const DefaultEndPoint: string = 'https://radar-backend.co.uk/'
 
-export const DefaultSchemaEndpoint =
-  'https://raw.githubusercontent.com/RADAR-base/RADAR-Schemas/add-audio/'
-export const DefaultSchemaSpecExtension = '.yml'
-export const DefaultSchemaSpecEndpoint =
-  DefaultSchemaEndpoint +
-  'specifications/active/aRMT-' +
-  DefaultSourceTypeRegistrationBody.sourceTypeCatalogVersion +
-  DefaultSchemaSpecExtension
-
-export const DefaultProtocolEndPoint: string = DefaultProtocolEndPointExport
-
-export const DefaultSourceProducerAndSecret: string = DefaultSourceProducerAndSecretExport
+// GITHUB SOURCES
+
+export const GIT_API_URI = 'https://api.github.com/repos'
+export const DefaultProject = 'RADAR-Base'
+
+export const DefaultProtocolRepo = 'RADAR-aRMT-protocols'
+// NOTE: Remove branch when merged
+export const DefaultProtocolBranch = '?ref=test'
+export const DefaultProtocolPath = 'protocol.json' + DefaultProtocolBranch
+export const DefaultProtocolEndPoint = [
+  GIT_API_URI,
+  DefaultProject,
+  DefaultProtocolRepo,
+  'contents'
+].join('/')
+
+export const DefaultSchemaRepo = 'RADAR-Schemas'
+// NOTE: Remove branch when merged
+export const DefaultSchemaBranch = '?ref=add-audio'
+export const DefaultSchemaSpecPath =
+  'specifications/active/aRMT-1.4.3.yml' + DefaultSchemaBranch
+export const DefaultSchemaSpecEndpoint = [
+  GIT_API_URI,
+  DefaultProject,
+  DefaultSchemaRepo,
+  'contents',
+  DefaultSchemaSpecPath
+].join('/')
```

Comment:
Please update the README as well as these are crucial configuration options. Also update the README and template for the secret.ts file as this has been moved from there.